tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -1,5 +1,16 @@
 {% extends "patterns/base_page.html" %}
-{% load wagtailcore_tags wagtailimages_tags static %}
+{% load wagtailcore_tags wagtailimages_tags util_tags static %}
+
+{% block extra_css %}
```

Comment:
btw, I believe the Work page also uses `StoryBlock` so can in theory have these

Reply:
Hmm, doesn't seem to work directly on the `work_page.html` - I guess it's because the streamfields are nested on this page and are not directly in `page.body`?

- work_page.html: https://github.com/torchbox/torchbox.com/blob/41b7d854f786203c53a9fa8535e5c4da8ae49fb2/tbx/project_styleguide/templates/patterns/pages/work/work_page.html#L40

- work_section_block.html: https://github.com/torchbox/torchbox.com/blob/41b7d854f786203c53a9fa8535e5c4da8ae49fb2/tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/work_section_block.html#L16

Reply:
it seems to work if you add the `extra_css` block in `work_page_base.html` 🤷🏼

Reply:
StoryBlock is used by:
- blog page
- standard page
- impact report page
- service page
- work page
- historical work page

We could add it to all of these, or possibly just add the check in the base template.

Reply:
From Helens debugging it seems that the streamfields are nested within Sections on the Work pages so the filter as it is right now doesn't work on work_base_page.html.

```
@register.filter(name="has_markdown_block")
def has_markdown_block(value):
    print('running')
    if not isinstance(value, StreamValue):
        return False

    print('still running')
    for block in value._raw_data:
        print(block["type"])
        if block["type"] == "markdown":
            print('found markdown')
            return True
```

On work pages this always prints 'Section' so I guess we'd need to adjust the filter to search within the sections rather than the `page.body`

Reply:
Helen helped out on Slack and the filter been updated to search nested streamfields within sections and it's now been added to `base.html`. 

Next up: add docs.

Reply:
@zerolab If you have a sec to cast an eye over the changes here it would be much appreciated! https://github.com/torchbox/torchbox.com/pull/295/commits/6e87ff6cd6223f03bd342e3ddc0559e70c3d56fa